src/etaoin/api.clj

```diff
@@ -2178,10 +2178,15 @@
 ;;
 
 (defn driver?
-  "Return true if `driver` is of `type` (e.g. on of: `:chrome`, `:edge`, `:firefox`, `:safari`)"
+  "Return true if `driver` is of `type` (e.g. one of: `:chrome`, `:edge`, `:firefox`, `:safari`)"
   [driver type]
   (= (dispatch-driver driver) type))
 
+(defn driver-type
+  "Returns the type of driver (e.g., `:chrome`, `:firefox`, `:safari`, or `:edge`)."
```

Comment:
Probably should add this to our toc in the ns docstring.

Reply:
Ah, yes, definitely. Will submit a patch here shortly.